core/tests/performance_tests/all_test.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
```

Comment:
At the moment, I don't think we should have this test. I think it might be more manageable if we still keep separate tests (like splash_test) but just refer to the variables defined in ``test_config`` to determine how long the load time should be or how big the total requests are. Generalizing tests can have unintended consequences. Tests should really be as specific as possible. The intention of making the tests declarative wasn't necessarily to generalize the running of them; I think it was to have a single source of truth declaring the performance characteristics of each page and thus make them easily configurable.